globalconf.go

```diff
@@ -98,14 +98,14 @@ func (g *GlobalConf) Delete(flagSetName, flagName string) error {
 	return nil
 }
 
-// Parses the config file for the provided flag set.
+// ParseSet parses the config file for the provided flag set.
 // If the flags are already set, values are overwritten
 // by the values in the config file. Defaults are not set
 // if the flag is not in the file.
 func (g *GlobalConf) ParseSet(flagSetName string, set *flag.FlagSet) {
```

Comment:
> Would you prefer I add tests proving it does what it says in the comment block?

Yes, please. I don't think this PR is where we should discuss ParseSet. It is beyond the scope. Could you please file another issue if you have more questions?